concordia/static/css/base.css

```diff
@@ -1083,6 +1083,22 @@ div.nav-secondary {
     box-shadow: inset 0 0.1875rem 0.3125rem rgba(0, 0, 0, 0.125);
 }
 
+/* 
+# Campaign Resources Panel
+*/
+
+div.related-links  {
+    background-color:#ddd;
+    border: #000 1px solid;
+    padding:1.5em;
```

Comment:
I would probably use the bootstrap `p-2` class for this for consistency but it's not a huge deal

Reply:
I'll put these notes in a separate cleanup ticket so they can be done when either of us is already in the CSS.